src/appkit.rs

```diff
@@ -282,6 +288,10 @@ impl NSApplication for id {
         msg_send![self, setServicesMenu:menu]
     }
 
+    unsafe fn setWindowsMenu_(self, menu: id) {
+        msg_send![self, setWindowsMenu:menu]
```

Comment:
The docs say that this was obsoleted in 10.10, in favor of the `windowsMenu` property. Is it possible to use that instead, or am I reading this code incorrectly?

The rest of this PR looks fine.